Doc/library/unittest.mock.rst

```diff
@@ -1308,9 +1308,23 @@ patch.dict
     :func:`patch.dict` can also be called with arbitrary keyword arguments to set
     values in the dictionary.
 
-    :func:`patch.dict` can be used as a context manager, decorator or class
-    decorator. When used as a class decorator :func:`patch.dict` honours
-    ``patch.TEST_PREFIX`` for choosing which methods to wrap.
+:func:`patch.dict` can be used as a context manager, decorator or class
+decorator. When used as a class decorator :func:`patch.dict` honours
+``patch.TEST_PREFIX`` for choosing which methods to wrap.
```

Comment:
I know this is just a copy & paste, but it would be nice to make the `patch.dict()` documentation self-contained and mention that the default value of `patch.TEST_PREFIX` is `test`. Or you could just link to https://docs.python.org/3/library/unittest.mock.html#test-prefix

Feel free to change other `patch.*` docs.

Reply:
Ok, I will copy & paste I think that is more useful have this in the same page.
 
> Feel free to change other `patch.*` docs.
Ok. I focused on patch.dict because I was working on https://github.com/python/cpython/pull/11437

After finish this PR with patch.dict, I will review whole patch.*  :-)